resources/authgear/templates/en/web/__error.html

```diff
@@ -131,13 +131,18 @@
     <li>{{ .details.msg }}</li>
     {{ else }}
     <li>{{ .Error.message }}</li>
-    {{ end }} {{ end }} {{ else if eq .Error.reason "UserNotFound" }} {{
-    template "__error_account" . }} {{ else if eq .Error.reason
-    "InvalidCredentials" }}
-    <li>
-      {{ template "error-invalid-credentials" (dict "AuthenticationType"
-      $info.AuthenticationType) }}
-    </li>
+    {{ end }} {{ end }}
+    {{ else if eq .Error.reason "UserNotFound" }}
+    {{ template "__error_account" . }}
+    {{ else if eq .Error.reason "InvalidCredentials" }}
+      {{ if eq $info.Channel "whatsapp" }}
+        <li>{{ template "error-invalid-whatsapp-otp" }}</li>
+      {{ else }}
+        <li>
+          {{ template "error-invalid-credentials" (dict "AuthenticationType"
```

Comment:
I suggest we simply remove the words "SMS" and "Email" to make the error message more generic. I think having the words "Whatsapp", "SMS", "Email" does not make the error message informative.